introduction_to_amazon_algorithms/linear_learner_abalone/Linear_Learner_Regression_csv_format.ipynb

```diff
@@ -13,15 +13,16 @@
     "## Contents\n",
```

Comment:
<p>Is the trailing period misplaced?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1709/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>No. I believe it just shows in a different line in reviewnb. Period is right next to format</p>